ui/src/config/section/compute.js

```diff
@@ -181,6 +181,10 @@ export default {
           docHelp: 'adminguide/virtual_machines.html#virtual-machine-snapshots',
           dataView: true,
           popup: true,
+          show: (record) => {
+            return ((['Running'].includes(record.state) && record.hypervisor !== 'LXC') ||
```

Comment:
@nvazquez @davidjumani @weizhouapache can you review the conditionals? cc @vladimirpetrov @borisstoyanov

Reply:
@rhtyd this is same as line 168 to 169. should be ok.